### PR TITLE
feat: final updates

### PR DESCRIPTION
**Pull request by Keith-CY**
1. remove period in headings
image
image
image
image
image
image
image
image
image
image
image
image

2. fix the live metrics
image
image
image

3. move cell background away from headings
image

4. Show `knowledge base` instead of `blogs` in navigation
image
image

### CODE REVIEW

File: src/server/api/routers/ckb.ts [hunk 1]
```diff
@@ -1,30 +1,26 @@
+import BigNumber from 'bignumber.js'
 import { explorerService } from '../../../services/ExplorerService'
+import { nodeService } from '../../../services/NodeService'
 import { createTRPCRouter, publicProcedure } from '../trpc'
 
+const DECIMAL = BigNumber(10 ** 8)
```
Comment (WhiteMinds) [inline src/server/api/routers/ckb.ts:6]: Not a good name, maybe it should be something like `ckbShannonPrecision` ?

File: src/services/NodeService/index.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class NodeService {
```
Comment (WhiteMinds) [inline src/services/NodeService/index.ts:1]: Maybe it would be better to call it `CKBNodeService`? Less misleading.

**Review by WhiteMinds — CHANGES_REQUESTED** [review]

File: public/locales/en/home.json [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
-    "text3": "Built on RISC-V and secured by Proof-of-Work, Common Knowledge Base is the most flexible and interoperable Layer 1 in the blockchain industry. It serves as the ultimate foundation of Nervos, a modular blockchain network built from the ground up to ensure outstanding security, decentralization, flexibility, and interoperability."
+    "text1": "Common Knowledge Base",
+    "text2": "Layer 1 Built for Modularity",
+    "text3": "Built on RISC-V and secured by Proof-of-Work, CKB is the most flexible and interoperable Layer 1 in the blockchain industry. It serves as the ultimate foundation of Nervos, a modular blockchain network built from the ground up to ensure outstanding security, decentralization, flexibility, and interoperability."
   },
   "slide_ckb_security": {
-    "text1": "Supreme Security and Decentrali&shy;zation.",
-    "text2": "Common Knowledge Base leverages the battle-tested Proof-of-Work consensus algorithm to ensure exceptional security and decentralization under all conditions. It is truly decentralized, permissionless, and uniquely resistant to capture by adversarial actors."
+    "text1": "Supreme Security and Decentralization",
```
Comment (WhiteMinds) [inline public/locales/en/home.json:9]:
```suggestion
    "text1_comment": "`&shy;` is required for automatic line feeds on mobile",
    "text1": "Supreme Security and Decentrali&shy;zation",
```

File: src/components/Footer/index.tsx [hunk 4]
```diff
@@ -38,11 +38,14 @@ export const Footer: FC<FooterProps> = props => {
     {
       title: 'ECOSYSTEM',
       links: [
-        { label: 'Nervos Foundation', url: 'https://www.nervos.org/' },
+        // { label: 'Nervos Foundation', url: 'https://www.nervos.org/' },
```
Comment (WhiteMinds) [inline src/components/Footer/index.tsx]:
```suggestion
```

Comment (Keith-CY) [inline src/components/Footer/index.tsx:41]:
```suggestion
```

File: src/components/Footer/index.tsx [hunk 5]
```diff
@@ -64,9 +67,9 @@ export const Footer: FC<FooterProps> = props => {
         </StyledLink>
       ),
       links: [
+        { label: 'Knowledge Base', url: '/knowledge-base' },
         /* not ready yet */
-        // { label: 'Knowledge Base', url: '/' },
-        { label: 'Blog', url: '/blogs' },
+        // { label: 'Blog', url: '/blogs' },
```
Comment (Keith-CY) [inline src/components/Footer/index.tsx:72]:
```suggestion
```

File: src/services/NodeService/index.ts (hunk 2 above)
Comment (Keith-CY) [reply src/services/NodeService/index.ts]: Updated by https://github.com/Magickbase/nervos-official-website/pull/84/commits/f58d610c8680320b4e1dd85b62b80f17723e6b3b

File: src/server/api/routers/ckb.ts (hunk 1 above)
Comment (Keith-CY) [reply src/server/api/routers/ckb.ts:6]: `DECIMAL` is a generally used term in blockchain context

File: src/pages/wallets/icons/index.tsx [hunk 6]
```diff
@@ -1,13 +1,16 @@
...
-import CkbBullIcon from './ckbBull.svg'
 import JoyIdIcon from './joyIdIcon.svg'
 import PortalWalletIcon from './portalWallet.svg'
 import SafePalIcon from './safePal.svg'
 import LedgerIcon from './ledger.svg'
 import OperaWalletIcon from './operaWallet.svg'
 import ImTokenIcon from './imToken.svg'
 
+const CkbBullIcon = () => (
+  <img src="/images/ckbull_logo.jpg" alt="ckbull" width="45" height="45" style={{ borderRadius: '50%' }} />
```
[the remark this answers is not in the document]
Comment (Keith-CY) [reply src/pages/wallets/icons/index.tsx:11]: The file is removed by https://github.com/Magickbase/nervos-official-website/pull/84/commits/3de73f43da7b6c20d61b7dc02bb2f5ecb0413c74

File: src/server/api/routers/ckb.ts (hunk 1 above)
Comment (WhiteMinds) [reply src/server/api/routers/ckb.ts:6]: Oh, I see.

**Review by WhiteMinds — APPROVED** [review]

**Pull request merged**